preview/d20/5e/Standard.htm.ftl

```diff
@@ -1753,6 +1753,37 @@ ${pcstring('ABILITYALL.Special Ability.VISIBLE.${specialQuality}.TYPE=SpecialQua
  </tr>
 <!-- STOP Feat Table -->
 
+<!-- START PFS Boon Table -->
```

Comment:
Unless 5th edition D&D has PFS Boons (which I'm not aware of) can we remove this change.